build_tools/azure/install.sh

```diff
@@ -84,6 +84,13 @@ elif [[ "$DISTRIB" == "conda-pip-latest" ]]; then
     setup_ccache
     python -m pip install -U pip
 
+    # NOTE(honles): scikit-image does not include pyproject.toml in its
+    #               current release and these are what it needs to build.
+    #               This can be removed when
+    #               https://github.com/scikit-image/scikit-image/pull/5016
```

Comment:
That MR has been merged: https://github.com/scikit-image/scikit-image/pull/5016

Reply:
Yes, but it has not been released yet. I will create a separate PR which can be merged after that is released to pypi/conda.

Reply:
I'm also trying to get Python 3.9 wheels built too, so hopefully that will all come together in the next release

See: https://github.com/scikit-image/scikit-image/pull/5052

Reply:
I would rather not build scikit-image from source for now. Maybe we can only install scikit-image for the builds that are for Python `PYTHON_VERSION < "3.9"` for now instead.

It's an optional dependency of scikit-learn, so it should still work.

Reply:
So would you prefer I change this to: `python -m pip install --only-binary :all: sckit-image || true`? So it only installs binary packages and only if they exists? I'd have to see if anything else has to be altered, but I'd prefer not to add config that has to be updated manually and just install it automatically if it can.